app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    command = command.split(" ")
```

Comment:
You can unpack command to three variables in one line. Also don’t use space in split as it use space by default
```suggestion
    command, old_file, new_file = command.split()
```